src/components/NavBar/navbar.tsx

```diff
@@ -0,0 +1,17 @@
+import { NavLink } from "react-router-dom";
+import useWalletConnect from "./connecttowallet";
+import './navbar.css';  
+
+export function NavBar() {
+  return (
+    <nav className="navbar-container">
+      <NavLink to ="/">
+        Marketplace
+      </NavLink>
+        {/* later replace with link to wallet connection */}
+        <div>{useWalletConnect()}</div>
+      <NavLink to="/CollectionsPage">View Assets</NavLink>
```

Comment:
Didn't find the <Switch> in the PR. Without it, the navigation doesn't work:
```suggestion
      <Switch>
          <Route exact path="/">
            ...
          </Route>
          <Route path="/CollectionsPage">
            <CollectionsPage />
          </Route>
        </Switch>
```
the route name should be in kebab case style 
https://www.theserverside.com/definition/Kebab-case